index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="CSS/style.css" />
+    <link
+      href="https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+      integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>Wasyihun-Portfolio</title>
+  </head>
+  <body>
+    <header class="navbar">
+      <div>
+        <a class="My_logo">MyLogo</a>
+      </div>
+      <nav class="menu_icon1">
+        <i class="bx bx-menu"></i>
+      </nav>
+    </header>
+    <section class="body_section">
+      <div class="body_section_title">
+        <h1>
+          I’m Wasyihun<br />
+          Glad to see<br />
+          you!
+        </h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <h2>LET’S CONNECT</h2>
+        <ul>
+          <li><i class="bx bxl-linkedin-square"></i></li>
+          <li><i class="bx bxl-github"></i></li>
+          <li><i class="bx bxl-twitter"></i></li>
+          <li><i class="bx bxl-facebook-square"></i></li>
+        </ul>
+        <ul>
+          <img src="image/Ellipse 3.svg" alt="Ellipse 3" class="Ellipse3" />
+          <img src="image/Ellipse 4.svg" alt="Ellipse 4" class="Ellipse4" />
+          <img src="image/Ellipse 5.svg" alt="Ellipse 5" class="Ellipse5" />
+          <img src="image/Ellipse 6.svg" alt="Ellipse 6" class="Ellipse6" />
+          <img src="image/Ellipse 7.svg" alt="Ellipse 7" class="Ellipse7" />
+          <img src="image/Ellipse 12.svg" alt="Ellipse 12" class="Ellipse12" />
+          <img src="image/Ellipse 13.svg" alt="Ellipse 13" class="Ellipse13" />
+          <img
+            src="image/Rectangle 4.svg"
+            alt="Rectangle 4"
+            class="Rectangle4"
+          />
+          <img
+            src="image/Rectangle 5.svg"
+            alt="Rectangle 5"
+            class="Rectangle5"
+          />
+          <img
+            src="image/Rectangle 7.svg"
+            alt="Rectangle 7"
+            class="Rectangle7"
+          />
+        </ul>
```

Comment:
- **[OPTIONAL]** Kindly add these images should be added like ☝🏽  as a background image in the CSS file.